paper/paper.Rmd

```diff
@@ -65,10 +65,12 @@ withr::local_options(list(
 
 # Statement of Need
 
-R is an interpreted, dynamically-typed programming language [@base2023]. It is a popular choice for statistical analysis and visualization, and is used by a wide range of researchers and data scientists. The `{lintr}` package is an open-source R package that provides static code analysis [@enwiki:1218663830] to check for a variety of common problems related to readability, efficiency, consistency, style, etc. In particular, by default it enforces the tidyverse style guide [@Wickham2023]. It is designed to be easy to use and integrate into existing workflows, and can be used as part of an automated build or continuous integration process. `{lintr}` also integrates with a number of popular IDEs and text editors, such as RStudio and Visual Studio Code, making it convenient for users to run `{lintr}` checks on their code as they work.
+A linter is a tool that analyzes code to identify potential errors, stylistic issues, or deviations from coding standards. It helps ensure consistency, readability, and best practices by flagging common mistakes, such as syntax errors, unused variables, or improper formatting. Linters are essential for improving code quality, preventing bugs, and maintaining a clean codebase, especially in collaborative development environments [@enwiki:1218663830]. `{lintr}` is an open-source package that provides linters for the R programming language, which is an interpreted, dynamically-typed programming language [@base2023], and is used by a wide range of researchers and data scientists. `{lintr}` can thus act as a valuable tool for R users to help improve the quality and reliability of their code.
 
 # Features
 
+By default, `{lintr}` enforces the tidyverse style guide [@Wickham2023,@Müller2024]. In this respect, it differs from other static code analysis tools in R (like `{codetools}` [@Tierney2024]), which are not opinionated and don't enforce any particular style of writing code, but, rather, check R code for possible problems (incidentally, `{lintr}` uses `{codetools}` as a backend for object usage linters). Additionally, `{lintr}` doesn't cover linting inline code for documentation generation (cf. `{roxylint}` [@Kelkhoff2024] for linting `{roxygen2}` comments [@Wickham2024roxy]).
```

Comment:
> Additionally, `{lintr}` doesn't cover linting inline code for documentation generation

This might make it seem like lintr would never throw on a roxygen2 comment, i.e. that it's out of scope, but it's just "not yet implemented". I think the scope overlap is slightly different, where {lintr} would only lint `@examples` code, but {roxylint} inspects _all_ `@tags`. Maybe something like

Additionally, {lintr} is concerned only with R code, so code-adjacent text like inline roxygen2 comments will not be covered (cf {roxylint}...)